extensions/datetime-range/README.md

```diff
@@ -1,8 +1,11 @@
 # Datetime Range Extension Specification (`dtr`)
```

Comment:
There's still the extension prefix in the header. Replace `dtr` with `-`

Reply:
Ahh. Good catch. Fixed